.github/workflows/pre-submit.actions.yml

```diff
@@ -17,13 +17,20 @@ jobs:
       - run: ./.github/workflows/scripts/pre-submit.actions/checkout.sh
 
   check-tscommon-tarball:
+    strategy:
+      matrix:
+        version: ["0.0.0", "0.1.0"]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
 
       - name: Untar the package tarball
         working-directory: .github/actions/tscommon
-        run: tar xvzf tscommon-0.0.0.tgz
+        env:
```

Comment:
Thought about this a bit more. Since we're checking out a single repo, extracting each tarball will fail as soon as we update the code, because each tarball is generated with a different codebase. Do I understand correctly?